source/Octopus.Tentacle.Tests.Integration/Support/IntegrationTest.cs

```diff
@@ -39,18 +33,31 @@ public void SetUp()
         public void TearDown()
         {
             Logger.Information("Tearing down");
-            if (cancellationTokenSource != null)
-            {
-                cancellationTokenSource.Dispose();
-                cancellationTokenSource = null;
-            }
+            
+            cancellationTokenRegistration?.Dispose();
```

Comment:
😵‍💫

Reply:
ugh, why do we even have to dispose this. I thought the CLR had a GC